APE12.rst

```diff
@@ -0,0 +1,188 @@
+Using Cookiecutter for the package-template
+-------------------------------------------
+
+author: Stuart Mumford, Thomas Robitaille
+
+date-created: 2016 November 17
+
+date-last-revised: 2016 November 17
+
+type: Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+This APE proposes migrating the `package-template
+<https://github.com/astropy/package-template/>`_ to use the `Cookiecutter
+<http://cookiecutter.readthedocs.io/>`_ project to provide templating facilities
+to the package template to ease use of the package template and to make it easy
```

Comment:
repeated 'package template' - maybe reword. Somewhere in abstract you need to spell out more clearly that creating new projects will be much easier for people (ease use doesn't quite cover it). Maybe have a separate sentence for the fact it's easier for people to use, and to 'subclass'. You said it better in the detailed bit. Can you try and improve it here too?